server/lib/currency.ts

```diff
@@ -269,3 +270,30 @@ export function reduceArrayToCurrency(array: AmountWithCurrencyAndDate[], curren
     },
   );
 }
+
+/**
+ * Gets exchange rate between given crypto currency and fiat.
+ */
+export async function getExchangeRate(fromCurrency: string, toCurrency: string): Promise<string> {
+  try {
+    const isFromCurrencyCrypto = SUPPORTED_CRYPTO_CURRENCIES.includes(fromCurrency);
+    const isToCurrencyCrypto = SUPPORTED_CRYPTO_CURRENCIES.includes(toCurrency);
+    const cryptoFromCurrencyToUSD = isFromCurrencyCrypto ? await getCryptoToUSDRate(fromCurrency) : undefined;
+    const cryptoToCurrencyToUSD = isToCurrencyCrypto ? await getCryptoToUSDRate(toCurrency) : undefined;
+
+    if (!isFromCurrencyCrypto && !isToCurrencyCrypto) {
+      return getFxRate(fromCurrency, toCurrency);
+    } else if (isFromCurrencyCrypto && !isToCurrencyCrypto) {
+      const toCurrencyUSDRate = await getFxRate(toCurrency, 'USD');
+      return cryptoFromCurrencyToUSD / toCurrencyUSDRate;
```

Comment:
Why use division instead of doing the opposite? It's the more natural way to use fx Rates.